pkg/didmethod/httpbinding/resolver.go

```diff
@@ -106,14 +107,19 @@ func New(endpointURL string, opts ...ResolverOpt) (*DIDResolver, error) {
 }
 
 // Read implements didresolver.DidMethod.Read interface (https://w3c-ccg.github.io/did-resolution/#resolving-input)
-func (res *DIDResolver) Read(DID string, versionID interface{}, versionTime string, noCache bool) ([]byte, error) {
+func (res *DIDResolver) Read(DID string, _ ...didresolver.ResolveOpt) ([]byte, error) {
```

Comment:
Does the universal resolver support these options?

Reply:
we will use them in the future

Reply:
> Does the universal resolver support these options?

Currently, there is no instruction on how to use these parameters in HTTP(s) binding in the [spec](https://w3c-ccg.github.io/did-resolution/#bindings).